src/Agent.Worker/JobRunner.cs

```diff
@@ -118,7 +118,8 @@ public sealed class JobRunner : AgentService, IJobRunner
                 {
                     Trace.Verbose($"Adding {taskInstance.DisplayName}.");
                     var taskRunner = HostContext.CreateService<ITaskRunner>();
-                    taskRunner.ExecutionContext = jobContext.CreateChild(taskInstance.InstanceId, taskInstance.DisplayName);
+                    var timeout = taskInstance.TimeoutInMinutes > 0 ? TimeSpan.FromMinutes(taskInstance.TimeoutInMinutes) : Timeout.InfiniteTimeSpan;
```

Comment:
@ericsciple: Presently taskInstance.TimeoutInMinutes is not available in DistributedTask.WebApi TaskInstance. We need to update DistributedTask.WebApi dll to get TimeoutInminutes. Till then can I put  "var timeout = Timeout.InfiniteTimeSpan"?. Once DistributedTask.WebApi dll updated I will add "var timeout = taskInstance.TimeoutInMinutes > 0 ? TimeSpan.FromMinutes(taskInstance.TimeoutInMinutes) : Timeout.InfiniteTimeSpan;".